codegenerator/cli/templates/static/codegen/src/eventFetching/chainWorkers/RpcWorker.res

```diff
@@ -83,15 +83,15 @@ let makeThrowingGetEventTransaction = (~getTransactionFields) => {
       | Some(fn) => fn
       // This is not super expensive, but don't want to do it on every event
       | None => {
-          transactionSchema->Utils.Schema.removeTypeValidationInPlace
+          let transactionSchema = transactionSchema->S.removeTypeValidation
```

Comment:
Is this now an immutable version of the function?

Reply:
Yep